### PR TITLE
Add MessageData for MessageHandlers

### PR DESCRIPTION
**Pull request by Seb-figment**

### CODE REVIEW

**Comment by pienkowb** [comment]
@Seb-figment You already added this [here](https://github.com/figment-networks/graph-ts/commit/552b10330994f59cb704f63520d1a5740e7f95ea) 🙂  I just moved it from `cosmos` to a feature branch (`message-handler`). We can merge it to `cosmos-v1.1` too, if you like.

File: chain/cosmos.ts [hunk 1]
```diff
@@ -58,6 +58,16 @@ export namespace cosmos {
...
   }
 
+  export class MessageData {
+    public message: Any
+    public block: HeaderOnlyBlock
+
+    constructor(message: Any, block: HeaderOnlyBlock) {
+      this.message = message
+      this.block = block
+    }
```
Comment (pienkowb) [inline chain/cosmos.ts:68]: This is the old syntax, let's use the latest version of `graph-proto-as-schema`.

Comment (pienkowb) [reply chain/cosmos.ts:68]: You can pull the master branch to see the new syntax.

Comment (Seb-figment) [reply chain/cosmos.ts]: @pienkowb I have merge master in

**Comment by lavabyrd** [comment]
@Seb-figment - this looks to contain Arweave again :(

**Comment by Seb-figment** [comment]
@lavabyrd cosmos-v1.1 have been updated with main.

**Review by lavabyrd — APPROVED** [review]
LGTM

**Pull request merged**